bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/HistoryStore.java

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Vector Informatik GmbH - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.ui.internal.findandreplace;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Vector;
+
+import org.eclipse.jface.dialogs.IDialogSettings;
+
+/**
+ * Store for search-histories or replace-histories inside of a dialog.. Stores
+ * the nodes using the DialogSettings-mechanism.
```

Comment:
```suggestion
 * Store for search histories or replace histories inside of a dialog. Stores
 * the nodes using the DialogSettings mechanism.
```

Reply:
Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/95c53c17f7afcd866e579b8f938a9b1781316588